src/mscorlib/shared/System/IO/PathInternal.cs

```diff
@@ -135,5 +140,13 @@ internal static void CheckSearchPattern(string searchPattern)
                 searchPattern = searchPattern.Substring(index + 2);
             }
         }
+
+        internal static string NormalizeDirectorySeparators(string path) => string.IsNullOrEmpty(path) ? path : NormalizeDirectorySeparatorsIfNeccessary(path) ?? path;
```

Comment:
Using IsNullOrEmpty here is not an improvement. A simple null check would be better because of it produces smaller code and works just fine.